clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -85,31 +99,37 @@
         "limit only applies to a single record in that case.";
 
     public static final String INDEX_INTERVAL_BYTES_CONFIG = "index.interval.bytes";
-    public static final String INDEX_INTERVAL_BYTES_DOCS = "This setting controls how frequently " +
+    public static final int INDEX_INTERVAL_BYTES_DEFAULT = 4096;
+    public static final String INDEX_INTERVAL_BYTES_DOC = "This setting controls how frequently " +
```

Comment:
I want to see this change gets merged since I love consistency. However, this change may break the compatibility as this field is public.